proposals/43:upgrade-11/legacy.sh

```diff
@@ -4,7 +4,6 @@
 source /usr/src/upgrade-test-scripts/env_setup.sh
 
 # Enable debugging
-set -x
 
```

Comment:
the "Enable debugging" comment stays? on purpose?
not a big deal...

Reply:
oops, will remove. (CI's not passing anyway)